File: lib/models/AccountHolder.dart
import 'package:cloud_firestore/cloud_firestore.dart';

class AccountHolder {
  AccountHolder(
      {required this.Country,
      required this.Email,
      required this.Image,
      required this.Money,
      required this.Name,
      required this.Password,
      required this.Phone,
      required this.IsActive,
      required this.Pin,
      this.CreatedAt = null,
      this.UpdatedAt = null});

  String Email;
  String Password;
  String Name;
  String Country;
  String Phone;
  String Pin;
  String Image;
  int Money;
  bool IsActive;
  Timestamp? CreatedAt;
  Timestamp? UpdatedAt;
}

File: Admin_Dashboard/Models/AccountHolder.cs
using Google.Cloud.Firestore;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Admin_Dashboard.Models
{
    [FirestoreData]
 public   class AccountHolder
    {
        public AccountHolder () { }
        public String Email { get; set; }
        public String Password { get; set; }
        public String Name { get; set; }
        public String Country { get; set; }
        public String Phone { get; set; }
        public String Pin { get; set; }
        public String Image { get; set; }
        public int Money { get; set; }
        public bool IsActive { get; set; }
        public Google.Cloud.Firestore.Timestamp createdAt { get; set; }
        public Google.Cloud.Firestore.Timestamp updatedAt { get; set; }
    }
}
